### PR TITLE
Improve grammar

### PR DESCRIPTION
**Pull request by jkeenan**
Per suggestion by oodler577 in https://github.com/Perl/perl5/pull/18736.

### CODE REVIEW

File: pod/perlnumber.pod [hunk 1]
```diff
@@ -129,10 +129,9 @@ used may lead to breakage of some of the above rules.
 =head1 Flavors of Perl numeric operations
 
 Perl operations which take a numeric argument treat that argument in one
-of four different ways: they may force it to one of the integer/floating/
-string formats, or they may behave differently depending on the format of
-the operand.  Forcing a numeric value to a particular format does not
-change the number stored in the value.
+of four different ways: they may force it to one of the integer, floating,
+or string formats; or they may behave differently depending on the format of
+the operand.
```
Comment (Grinnz) [inline pod/perlnumber.pod]: Second sentence got lost

Comment (jkeenan) [reply pod/perlnumber.pod]: Restored. Thanks.

**Comment by khwilliamson** [comment]
I think this can be merged after squashing

**Comment by jkeenan** [comment]
> I think this can be merged after squashing

Squashed, rebased and merged to blead in ffa5817eaa.  Closing ticket.

**Pull request closed without merging**